README.md

```diff
@@ -5,16 +5,14 @@
 The Tuva Project is a collection of data marts and terminology code sets that transform healthcare data for analytics.
 
 **Data Marts:**
```

Comment:
I think we need to add a section to the readme that goes over how the data mart vars work and that they are disabled by default. The instructions are buried in the integration_tests folder so it's easy to miss.